buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1274,7 +1276,7 @@ class BeamModulePlugin implements Plugin<Project> {
         // has different dependencies than our project.
         if (config.getName() != "errorprone" && !inDependencyUpdates) {
           config.resolutionStrategy {
-            force project.library.java.values()
+            force project.library.java.values().findAll { it.split(':').size() > 2 }
```

Comment:
`force project.library.java.values()` does not take GCP libraries BOM into account. So excluding the versionless coordinates.